skt/__init__.py

```diff
@@ -484,13 +491,18 @@ def prepare(self, clean=True):
 
         shutil.copyfile(self.basecfg, self.get_cfgpath())
 
+        args = self.defmakeargs + [self.cfgtype]
+        logging.info("prepare config: %s", args)
+        subprocess.check_call(args)
+        self._ready = 1
+
```

Comment:
I think you might have forgotten to remove this from below, when moving it, so now we would have two of these. Or am I missing something?

Reply:
Whoops, that got introduced in all the rebases. Removed.